src/presentational-components/shared/breadcrubms.js

```diff
@@ -1,17 +1,19 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { NavLink } from 'react-router-dom';
 import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
 
 const RbacBreadcrumbs = (breadcrumbs) => {
   return (
-    <Breadcrumb style={ { minHeight: 18 } }>
-      { breadcrumbs && Object.values(breadcrumbs).map(item => (
-        <BreadcrumbItem key={ item.title } isActive={ item.isActive }>
-          { (item.to && <NavLink exact to={ item.to }>{ item.title }</NavLink>) || item.title }
-        </BreadcrumbItem>
-      )) }
-    </Breadcrumb>
+    <Fragment>
```

Comment:
Why is the fragment needed? I see only one element inside.

If that because of the condition? If so, I'd move the condition outside.

Reply:
:+1:

Reply:
Fragment removed